hashicorp/waypoint
# ui: modernize `<AppBreadcrumbs>`

## Why the change?

Helps us clear some linting issues and keep all our components consistent and familiar.

## Any review notes?

I recommend [hiding whitespace changes](https://github.com/hashicorp/waypoint/pull/2164/files?diff=split&w=1).

## How do I test it?

1. Check out the branch
2. Boot the UI dev server (`cd ui && yarn start`)
3. [Visit http://localhost:4200](http://localhost:4200)
4. Browse around a bit
5. Verify that the breadcrumbs are behaving themselves
